apps/server/src/modules/authentication/controllers/api-test/login.api.spec.ts

```diff
@@ -144,6 +144,30 @@ describe('Login Controller (api)', () => {
 				await request(app.getHttpServer()).post(`${basePath}/local`).send(params).expect(401);
 			});
 		});
+		describe('when user login fails cause account is deactivated', () => {
+			const setup = async () => {
+				const newUser: User = userFactory.buildWithId();
+				const deactivatedAccount: AccountEntity = accountFactory.buildWithId({
+					userId: newUser.id,
+					username: newUser.email,
+					password: defaultPasswordHash,
+					deactivatedAt: new Date(),
```

Comment:
it just occured to me, that if date is in future, it should still allowed the login
this would be another case to be tested
when we have here new Date(), and the check below executes just a few millisecond after, yes it works, because we use timestamp, but it would be better to clearly define a date in the past and one in the future.